build/testdata/065.in

```diff
@@ -0,0 +1,4 @@
+x = 123
+y = 0123
+z = 0
+foo(bar=[0, 01, 02, 3])
```

Comment:
Add tests: `0o12`, `0x12`

Reply:
0o12 is already covered by idempotency tests.